blocks/form/form.js

```diff
@@ -367,6 +368,7 @@ async function createForm(formURL) {
   });
   groupFieldsByFieldSet(form);
   decorateFile(form);
+  if (config.layout === 'wizard') { decorateWizard(form); }
```

Comment:
Just wanted to understand why not using variations ?

Reply:
should't wizard be loaded only when required ?

Reply:
done